Scripts/fastlane/Fastfile

```diff
@@ -190,6 +201,72 @@ import "./ScreenshotFastfile"
     build_and_upload_itc(skip_prechecks: true, skip_confirm: options[:skip_confirm])
   end
 
+  #####################################################################################
+  # build_and_upload_installable_build
+  # -----------------------------------------------------------------------------------
+  # This lane builds the app and upload it for adhoc testing 
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane build_and_upload_installable_build [version_long:<version_long>]
+  #
+  # Example:
+  # bundle exec fastlane build_and_upload_installable_build 
+  # bundle exec fastlane build_and_upload_installable_build build_version:123 
+  #####################################################################################
+  desc "Builds and uploads an installable build"
+  lane :build_and_upload_installable_build do | options |
+    alpha_code_signing
+
+    # Get the current build version, and update it if needed
+    version_config_path = "../../config/Version.internal.xcconfig"
+    versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
+    build_version = versions["VERSION_LONG"]
+
+    if options.key?(:build_version)
+      build_version = options[:build_version]
+
+      UI.message("Updating build version to #{build_version}")
+
+      versions["VERSION_LONG"] = build_version
+      new_config = Xcodeproj::Config.new(versions)
+      new_config.save_as(Pathname.new(version_config_path))
+    end
+
+    gym(
+      scheme: "WordPress Alpha",
+      workspace: "../WordPress.xcworkspace",
+      export_method: "enterprise",
+      clean: true,
+      output_directory: "../build/",
+      derived_data_path: "../derived-data/alpha/",
+      export_team_id: ENV["INT_EXPORT_TEAM_ID"],
+      export_options: { method: "enterprise" })
+
+    sh("mv ../../build/WordPress.ipa \"../../build/WordPress Alpha.ipa\"")
+      
+    hockey(
```

Comment:
This will need to be changed when we migrate away from HockeyApp but it should be straightforward to update here as well as for internal builds.